lambda/translprim.ml

```diff
@@ -1472,7 +1472,7 @@ let layout_of_ty_for_idx_set env loc ty =
     thing. *)
   let jkind = Ctype.type_jkind env ty in
   let mbe = Typedecl.mixed_block_element env ty jkind in
-  let mbe = transl_mixed_block_element env (to_location loc) ty mbe in
+  let mbe = Typeopt.transl_mixed_block_element env (to_location loc) ty mbe in
```

Comment:
Once you remove `transl_mixed_block_element` from `lambda.mli`, this diff is probably unneeded.